editor/editor_sectioned_inspector.cpp

```diff
@@ -29,7 +29,7 @@
 /*************************************************************************/
 
 #include "editor_sectioned_inspector.h"
-
```

Comment:
Keep this line between the class declaration header and the rest: https://docs.godotengine.org/en/latest/community/contributing/code_style_guidelines.html#header-includes